src/NHSD.BuyingCatalogue.Identity.Api/ViewModels/Account/ResetPasswordViewModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.ViewModels.Account
+{
+    public sealed class ResetPasswordViewModel
+    {
+        [DataType(DataType.Password)]
+        [PasswordValidation(ErrorMessage = ErrorMessages.PasswordConditionsNotMet)]
+        [DisplayName("Enter a password")]
+        public string Password { get; set; }
+
+        [DataType(DataType.Password)]
+        [Compare("Password", ErrorMessage = ErrorMessages.PasswordMismatch)]
+        [DisplayName("Confirm password")]
+        public string ConfirmPassword { get; set; }
+
+        internal static class ErrorMessages
+        {
+            internal const string PasswordConditionsNotMet = "The password you’ve entered does not meet the criteria";
+            internal const string PasswordMismatch = "Passwords do not match";
+        }
+
+        [AttributeUsage(AttributeTargets.Property, AllowMultiple = true, Inherited = false)]
+        internal class PasswordValidation : ValidationAttribute
```

Comment:
Does this class need to be `internal`? Unless we need to reference it in tests can it be made `private`?